examples/pxScene2d/src/shell.js

```diff
@@ -26,14 +26,59 @@ px.import({ scene: 'px:scene.1.js',
   var scene = imports.scene;
   var keys  = imports.keys;
 
+  var Logger = function (logLevel) {
```

Comment:
Why don't you use an existing Logger from rcvrcore?

Reply:
added another implementation based on Logger.js